### PR TITLE
fix DEVICE_LOCATION_RELATIONSHIP _key and _type; add test

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: src/steps/fetch-hardware/index.ts [hunk 1]
```diff
@@ -8,14 +8,19 @@ import {
...
   mapHardwareRelationship,
   mapHardwareLocationRelationship,
 } from '../../converter';
 
 const step: IntegrationStep = {
   id: 'fetch-hardware',
   name: 'Fetch Snipe-IT listing of hardware assets',
-  types: ['hardware', DEVICE_MANAGEMENT_RELATIONSHIP],
+  types: [
+    'hardware', // Deprecated. Step creates mapped entities intead.
```
Comment (aiwilliams) [inline src/steps/fetch-hardware/index.ts:20]: I guess you'll keep this around until another PR that stops producing the `hardware` entities?

Comment (erkangz) [reply src/steps/fetch-hardware/index.ts:20]: yep
